docs/api.md

```diff
@@ -544,6 +549,8 @@ Page is guaranteed to have a main frame which persists during navigations.
     - `left` <[string]> Left margin, accepts values labeled with units.
 - returns: <[Promise]<[Buffer]>> Promise which resolves with PDF buffer.
 
+`page.pdf()` generates a pdf of the page with `print` css media. To get a pdf with `screen` media, use [page.emulateMedia(mediaType)](#pageemulatemediamediatype).
```

Comment:
To get a pdf -> To generate a pdf

use page.emulateMedia -> call page.emulateMedia before calling page.pdf().